tag models done

Manit1 opened the pull request:
Fixes #19

Manit1 committed:
tag models done

Manit1 committed:
question update added

Manit1 committed:
add delete

abhishek97 commented:
@Manit1 
- Saving Tags on questions doesn't work.
- We don't allow deleting tags

Manit1 committed:
question fix

Manit1 commented:
Hey @abhishek97,
I fixed the issue with saving tags on questions and removed the delete route but still not able to figure how can I implement filtering with tags.

Basically I need to do something like
```js
DB.questions.findOne({
    where: {
        id: 1,
    },
    include: {
        model: DB.tags,
        where: {
            name: {
                $iLike: '%something%'
            }
        },
        required: false
    }
}).then(o => console.log(o.tags))
```
to filter this but not able to think of how to integrate this in base controller
